### DISCUSSION TITLE
Zweck dieses Repos

### DISCUSSION THREAD

**Discussion by jazznbass** [jazznbass/aesf_homepage#2] [General]
@jannisbosch 
@researchtool

Ich habe überlegt, dass wir eine kleine Homepage für die AESF aufziehen können.

Als basis bietet sich github pages mit Jeykll an. Ich habe hier eine Standardseite eingerichtet.
Neue Einträge lassen sich von jedem Nutzer des Repos als Post mit wenig aufwand online ergänzen.

Ich dachte, dass man von hier z.B. die Mailingliste verlinken kann, sowie das OSF Project. Auch Infos über die nächste Tagung lassen sich dort ablegen.

Die Seite soll praktisch niemandem speziell gehören, sondern von der Community gepflegt werden. Also von einem github repro aus. 

Ich dachte, dass wir eine solch Seite erstmal aufstellen können und dann die Leute der AESF hinzuziehen, wenn erstmal so alles steht.

Was denkt Ihr? Sollen wir das angehen und gibt es ggf. noch jemanden, den man in dieser Phase gut einbinden kann?

**Comment by jazznbass** [comment]
Homepage als html wird mit jedem commit automatisch gerendert und ist hier:

https://jazznbass.github.io/aesf_homepage/

**Comment by OterRed** [comment]
Super Sache, vielen Dank Jürgen für diese Initiative!
